### PR TITLE
[Documentation] Architecture - Divisions

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/architecture.rst [hunk 1]
```diff
@@ -11,6 +11,65 @@ All bundles share the same conventions for naming things and the way of data per
 
 For deeper understanding of how Doctrine works, please refer to the `excellent documentation on their official website <http://doctrine-orm.readthedocs.org/en/latest/>`_.
 
+Division into Components, Bundles, Platform
+-------------------------------------------
+
+Components
+''''''''''
```
Comment (pamil) [inline docs/book/architecture.rst]: Shouldn't it be `~~~`?

File: docs/book/architecture.rst [hunk 2]
```diff
@@ -11,6 +11,65 @@ All bundles share the same conventions for naming things and the way of data per
...
+
+Every single component of Sylius can be used standalone. Taking the ``Taxation`` component as an example,
+it's only responsibility is to calculate taxes,it does not matter whether these will be taxes for products or anything else, it is fully decoupled.
```
Comment (michalmarcinkowski) [inline docs/book/architecture.rst]: Missing space after comma `...taxes,it does not...`

File: docs/book/architecture.rst [hunk 3]
```diff
@@ -11,6 +11,65 @@ All bundles share the same conventions for naming things and the way of data per
...
+In order to let the Taxation component operate on your objects you need to have them implementing the ``TaxableInterface``.
+Since then they can have taxes calculated.
+Such approach is true for every component of Sylius.
+Besides components that are strictly connected to the e-commerce needs, we have plenty of components that are more general. For instance Attribute, Mailer, Locale etc.
```
Comment (michalmarcinkowski) [inline docs/book/architecture.rst]: Link to `Components` section?

File: docs/book/architecture.rst [hunk 4]
```diff
@@ -11,6 +11,65 @@ All bundles share the same conventions for naming things and the way of data per
...
+
+All the components are packages available via `Packagist <https://packagist.org/>`_.
+
+Bundles
+'''''''
+
+These are the Symfony Bundles - therefore if you are a Symfony Developer, and you would like to use the Taxation component in your system,
+but you do not want to spend time on configuring forms or services in the container. You can include the ``TaxationBundle`` in your application
+with minimal or even no configuration to have access to all the services, models, configure tax rates, tax categories and use that for any taxes you will need.
+
```
Comment (michalmarcinkowski) [inline docs/book/architecture.rst]: Link to `Using bundles`

File: docs/book/architecture.rst [hunk 5]
```diff
@@ -11,6 +11,65 @@ All bundles share the same conventions for naming things and the way of data per
...
+Platform
+''''''''
+
+This is a fullstack Symfony Application, based on Symfony Standard. Sylius Platform gives you the classic, quite feature rich webshop.
+Before you start using Sylius you will need to decide whether you will need a full platform with all the features we provide, or maybe you will use decoupled bundles and components
+to build something very custom, maybe smaller, with different features.
+But of course the platform is highly customizable to fit different bussiness models.
```
Comment (michalmarcinkowski) [inline docs/book/architecture.rst]: How about something like "... the platform itself is highly flexible and can be easily customized to meet all business requirements you may have."?

**Comment by pjedrzejewski** [comment]
Thanks Magda! 👍

**Pull request merged**